upp/configs/variables.yaml

```diff
@@ -8,16 +8,10 @@ jets:
     - HadronConeExclTruthLabelPt
     - HadronConeExclTruthLabelLxy
     - HadronConeExclTruthLabelDR
-    - HadronGhostTruthLabelID
-    - HadronGhostExtendedTruthLabelID
-    - HadronGhostTruthLabelPt
-    - HadronGhostTruthLabelLxy
-    - HadronGhostTruthLabelDR
     - pt
     - eta
     - mass
     - n_tracks
-    - n_truth_promptLepton
```

Comment:
Is there a reason why you remove these variables from the "general" variables config? I guess you don't really use them so therefore, could you undo the changes?

Reply:
The `HadronGhostTruth` labels were added to the TDD recently and are available in the latest central dump. Therefore, they should stay in the config.

The `n_truth_promptLepton` variable is used in the GN3 configs downstream in salt, therefore the current "baseline" config is the `GN3_dev.json` from the TDD. In general, the default configs in UPP are kept up-to-date for the latest tagger available (GN3 in this case).

Therefore, I would keep them in the default config. Different use-cases can have different configs, but I would rather avoid restructuring them in this PR.